### PR TITLE
Gtw 29 sign up user account connection

### PR DESCRIPTION
**Pull request by R11manish**
# [PR Name]

## Type of PR

- [] - feature
- [] - bugfix
- [] - chore
- [] - docs

## Description

Lorem Ipsum

## Checklist

- [] - Provided useful description for this PR
- [] - Provided the root cause of this bugfix
- [] - Tested on supported devices

### CODE REVIEW

File: src/components/atoms/social-auth-card.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  return (
+    <Paper
+      sx={{
+        border: 1,
+        padding: 3.5,
+        borderRadius: 2,
+        width: '100%',
+        height: 223,
+        borderColor: '#E5E5E51F',
```
Comment (andrefelizardo) [inline src/components/atoms/social-auth-card.tsx:53]: Could you check if we have this color as a variable?

File: src/components/features/signup/sections/signup-methods.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            }}
+          >
+            {method.methodName}
+          </Button>
+        ))}
+      </Stack>
+      <Typography color="text.secondary">
+        By continuing you agree to our{' '}
+        <Link href="/terms" underline="none">
+          Terms of Service{' '}
```
Comment (andrefelizardo) [inline src/components/features/signup/sections/signup-methods.tsx:118]: Hardcoded text

File: src/components/features/signup/signup.tsx [hunk 3]
```diff
@@ -92,8 +96,10 @@ export function Signup() {
     {
       onSuccess() {
         enqueueSnackbar(t('form.profile-created'));
-        setProfileCreated(true);
-        onInvalidateMe();
+        // change the code to make it dynamically: here done this just to show the flow
+        setShowConnectMoreAuthDialog(true);
+        // setProfileCreated(true);
+        // onInvalidateMe();
```
Comment (andrefelizardo) [inline src/components/features/signup/signup.tsx:102]: Comment

File: src/components/features/signup/utlis/connect-more-auth-dialog.tsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+          variant="h4"
+          width={{ xs: '100%', md: '35%' }}
+          mt={{ xs: 4, md: 20.75 }}
+          gutterBottom
+        >
+          Connect accounts to make it easier to find you on Gateway and earn
+          credentials
+        </Typography>
+        <Typography width={{ xs: '100%', md: '35%' }} variant="body1">
+          Connect your social media, wallets, and other platform accounts to
```
Comment (andrefelizardo) [inline src/components/features/signup/utlis/connect-more-auth-dialog.tsx:80]: Hardcoded text

File: src/components/features/signup/utlis/connect-more-auth-dialog.tsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+            connectHandler={twitterConnection}
+          />
+        </Stack>
+        <Typography
+          width={{ xs: '100%', md: '35%' }}
+          variant="caption"
+          mt={4}
+          gutterBottom
+        >
+          We never share any of your data with third parties, and we only
```
Comment (andrefelizardo) [inline src/components/features/signup/utlis/connect-more-auth-dialog.tsx:118]: Hardcoded text

**Review by andrefelizardo — CHANGES_REQUESTED** [review]

**Review by NunoMartins21 — APPROVED** [review]

**Review by andrefelizardo — APPROVED** [review]

**Pull request merged**